ui/src/me/components/account/ViewTokenOverlay.tsx

```diff
@@ -39,20 +44,20 @@ export default class ViewTokenOverlay extends PureComponent<Props> {
             mode={PermissionsWidgetMode.Read}
             heightPixels={500}
           >
-            {permissions.map((p, i) => {
+            {Object.keys(obj).map((p, i) => {
```

Comment:
`Object.entries` could be helpful here:

```suggestion
            {Object.entries(obj).map(([type, permissions]) => {
```

then you don't need to index back into `obj` later

Reply:
I don't want to loop through an array of type then array of permissions. So that's why I am looping on the keys only.